### PR TITLE
feat(cb2-8711): add edit tech record button to bottom of tech record

### PR DESCRIPTION
**Pull request by emilyrkelly21**
## Submit button at the bottom of Tech Record

add edit tech record button to bottom of tech record
[CB2-8711](https://dvsa.atlassian.net/browse/CB2-8711)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge

### CODE REVIEW

**Review by gjulien-bjss — DISMISSED** [review]

**Review by shivangidas — DISMISSED** [review]

File: src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <ng-container *ngIf="(isArchived$ | async) === false">
-  <app-button *ngIf="!isEditing; else editing" id="edit" (clicked)="checkIfEditableReasonRequired()">Amend</app-button>
+  <app-button *ngIf="!isEditing; else editing" [id]="customId + 'edit'" (clicked)="checkIfEditableReasonRequired()">Amend</app-button>
```
Comment (naathanbrown) [inline src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html:2]: I personally think working customId's in is overkill. Just 'BottomEdit' I think would be fine? Also on this app-button idk if I am being silly and missing something but where is customId defined?

Comment (gjulien-bjss) [reply src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html:2]: It's defined in the `.ts` of the component?

Comment (gjulien-bjss) [reply src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html:2]: We want ids to be unique, there's two of those components of the page. How else would we get unique ids on that page?

Comment (naathanbrown) [reply src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html:2]: Yeah I was being silly! I gave myself the out 😂

**Review by naathanbrown — APPROVED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Review by gjulien-bjss — APPROVED** [review]

**Review by shivangidas — APPROVED** [review]

**Pull request merged**